cpp/include/cudf/strings/string_view.hpp

```diff
@@ -314,7 +307,7 @@ class string_view {
   /**
    * @brief Default constructor represents an empty string.
    */
-  CUDA_HOST_DEVICE_CALLABLE string_view() : _data(""), _bytes(0), _length(0), _char_width(0) {}
+  CUDA_HOST_DEVICE_CALLABLE string_view() : _data(""), _bytes(0), _length(0) {}
```

Comment:
What does `_data("")` do here? `""` is a temporary, and `_data` will end up pointing to it, which should be invalid as soon as we exit the constructor. I must be missing something.

Reply:
The `""` is a constant. It is intended to ensure the default ctor creates an instance representing an empty string and not a nullptr. This is just a convenience for use in some `thrust` and other utility functions.